opentracing-spring-cloud/pom.xml

```diff
@@ -73,6 +73,11 @@
       <artifactId>opentracing-spring-messaging-starter</artifactId>
     </dependency>
 
+    <dependency>
+      <groupId>io.opentracing.contrib</groupId>
+      <artifactId>opentracing-mongo-driver</artifactId>
```

Comment:
Could you please exclude all transitive dependencies? This library should not pull any 3rd party deps other than instrumentations. 

There are also dependency tests to ensure it.